packages/imperative/src/Imperative.ts

```diff
@@ -365,7 +365,7 @@ export class Imperative {
      * Parse command line arguments and issue the user's specified command
      * @returns {Imperative} this, for chaining syntax
      */
-    public static parse(args?: string, context?: IYargsContext): Imperative {
+    public static parse(args?: string | string[], context?: IYargsContext): Imperative {
```

Comment:
This may be considered a breaking change for TS CLI developers.
As an example: VScode's `TreeItem.label` property changed from `string` to `string | TreeItemLabel` and some extensions had to adjust to the change eventually.